docs/customization/fixture_suites.rst

```diff
@@ -1,59 +1,59 @@
 Customizing Fixture Suites
 ==========================
 
-What are suites?
-~~~~~~~~~~~~~~~~
-Suites are pockets of fixtures that can be run together.
+What are fixture suites?
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-Why would you customize a suites?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-If you need to create a configuration file where you define all necessarily configuration to your store.
+Suites are predefined groups of fixtures that can be run together. They can be for example full shop configurations for manual tests purposes.
+
+Why would you customize fixture suites?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    * tailoring the default Sylius fixture suite to your needs (removing Orders for example)
+    * creating your own fixture suite
```

Comment:
```suggestion
    * creating your own fixture suite
    * adjusting currencies or languages to present shop for customer
```

Reply:
this is covered by the first dot :)
`tailoring the default Sylius fixture suite to your needs (removing Orders for example)`